### DISCUSSION TITLE
AR5700D support

### DISCUSSION THREAD

**Discussion by autodiag** [AlexandreRouma/SDRPlusPlus#1337] [Ideas]
hello
possible add support source to AOR AR5700D ?
this IQ data out
SDR Sharp newest version have it this now
work great,but SDR++ is better :D ..

**Comment by AlexandreRouma** [comment]
I don't have one but if they're willing to send me one sure.

**Comment by autodiag** [comment]
sending a receiver is a bit utopian :/
but I can think of a remote access or something like that

I can send SDRSharp with AOR IQ support

**Reply by AlexandreRouma** [reply]
not really, almost all other SDR manufacturers have sent dev units for the project, see the hardware donor list: 
![image](https://github.com/AlexandreRouma/SDRPlusPlus/assets/17126412/f6651866-e2dd-4ec5-97b9-912ead49013d)

Remote hardware is just not sufficient for development. Ask any SDR software dev...

**Comment by autodiag** [comment]
ok
that's a good idea but AOR is a simple Japanese who won't even listen to the description of the community protocol of basic control, so radio :D
